src/yaml/pull_parser.cr

```diff
@@ -56,6 +56,8 @@ class YAML::PullParser
       ptr = @event.data.sequence_start.tag
     when .scalar?
       ptr = @event.data.scalar.tag
+    else
```

Comment:
I've found another ocurrence of this in line 226

```
In src/yaml/pull_parser.cr:226:5

 226 | case kind
       ^
Warning: case is not exhaustive for enum YAML::EventKind.

Missing members:
 - NONE
 - STREAM_START
 - STREAM_END
 - DOCUMENT_START
 - DOCUMENT_END
 - SEQUENCE_END
 - MAPPING_END
```

Reply:
Good catch! How did you find it? It seems it wasn't used in the code.

I just pushed a commit with specs for it. And I also handled a couple more cases. Thank you exhaustive case check! 😛

Reply:
I'm checking this against test-ecosystem together for the 0.34 release ...

And thanks to other PR related to warnings, this is easier to depict now.